frontend/simple/views/components/Graphs/SupportHistory.vue

```diff
@@ -1,7 +1,6 @@
 <template>
-  <section>
-    <h3 class="title"><i18n>Support History</i18n></h3>
-    <p v-if="history.length === 0">Your group is still in its first month.</p>
+  <dashboard-section title="Support History">
```

Comment:
how about moving the `dashboard-section` to GroupDashboard.vue? that way we could make these section components dashboard independent altogether

Reply:
What do you exactly mean by moving to GroupDashboard?

Reply:
yes! exactly! this way we can reuse these components elsewhere, outside of the dashboard too